### ISSUE TITLE
B&S ImpliedVolatilty for a put

### ISSUE THREAD

**Issue by loogies06** [finmath/finmath-lib#19]
Hello,

Is it possible to have implied volatilty for a put ?
 AnalyticFormulas.blackScholesOptionImpliedVolatility

Currently it is only available for a call.

Thanks,
Luc

**Comment by cfries** [comment]
Yes. Until then: You may use put call parity first, then call the function.
